lib/fp.js

```diff
@@ -174,6 +174,26 @@ const restLeft = fn => (...spreadArgs) => {
   fn(args, ...namedArgs, callback);
 };
 
+// Identity function
+//   x - any argument
+// Result: argument x
+const id = x => x;
+
+// Async identity function
+//   x - any argument
+//   callback - function, (err, data)
+const asyncId = (x, callback) => {
+  callback(null, x);
+};
+
+// Mapable object iteration identity function
+//   items - array
+//   fn - function
+//   callback - function, (err, data)
+const mapIterationId = (items, fn, callback) => {
+  callback(null, items.map(items));
```

Comment:
```suggestion
  callback(null, items.map(fn));
```

Reply:
It's identify iteration function. So, I think, for every element we have call identify function